cmd/cosign/cli/options/attest.go

```diff
@@ -75,4 +76,7 @@ func (o *AttestOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().BoolVarP(&o.SkipConfirmation, "yes", "y", false,
 		"skip confirmation prompts for non-destructive operations")
+
+	cmd.Flags().BoolVar(&o.TlogUpload, "tlog_upload", true,
```

Comment:
i think it should match the other flags with composed name

for example the `--no-upload` then should be `--tlog-upload`

Reply:
thanks for the comment.
renamed to `--no-tlog-upload`, as the `--upload` flag refers to uploading signatures and not tlog.

Reply:
thank you!